behave_framework/src/minifi_test_framework/steps/core_steps.py

```diff
@@ -0,0 +1,56 @@
+#
+#  Licensed to the Apache Software Foundation (ASF) under one or more
+#  contributor license agreements.  See the NOTICE file distributed with
+#  this work for additional information regarding copyright ownership.
+#  The ASF licenses this file to You under the Apache License, Version 2.0
+#  (the "License"); you may not use this file except in compliance with
+#  the License.  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+#
+
+import logging
+import random
+import string
+
+import humanfriendly
+from behave import when, step
+from minifi_test_framework.containers.directory import Directory
+from minifi_test_framework.containers.file import File
+from minifi_test_framework.core.minifi_test_context import MinifiTestContext
+
+
+@when("both instances start up")
+@when("all instances start up")
+def step_impl(context: MinifiTestContext):
+    for container in context.containers:
+        assert container.deploy()
+    assert context.minifi_container.deploy() or context.minifi_container.get_logs()
```

Comment:
Here the `context.minifi_container.get_logs()` only returns the log string but does not print the logs, I suppose this should be `log_app_output`